May need Docker logout before build

karlinhigh opened the pull request:
I am new to Docker, and might not be understanding something here. I just installed Docker on Windows 10 Pro 1803 64-bit. My first effort at "docker-compose build" got me a message that said...

ERROR: Service 'lilypond-build' failed to build: Get https://registry-1.docker.io/v2/library/ubuntu/manifests/18.04: unauthorized: incorrect username or password

A web search advised me to log out my account from Docker, and the error was resolved. Maybe LilyDev's Docker instructions should mention this? Not sure which step would be best to include it.

fedelibre commented:
Thanks, nice addition. But I have some comments:

1. Unfortunately [the stable documentation on live.debian.net is currently for an oldstable](https://lists.debian.org/debian-live/2015/07/msg00030.html) (version 3), but we are using the last stable (version 4) for jessie. I'm going to add instructions on how to build the stable live-manual.

2. I've never used it before but IIUC a binary hook should change files in the build tree (not files in /etc or you'll modify the system running the build). From the documentation:

    >The commands in your hook do not run in the chroot, so take care to not modify any files outside of the build tree, or you may damage your build system!

I think that you should modify the files in the binary/ directory but you don't have much there, just some .deb packages (not nano). I think that this is what you can modify in binary/:

```
├── install
│   ├── gtk
│   │   ├── initrd.gz
│   │   ├── install.bat
│   │   └── vmlinuz
│   ├── initrd.gz
│   ├── install.bat
│   └── vmlinuz
├── isolinux
│   ├── advanced.cfg
│   ├── hdt.c32
│   ├── install.cfg
│   ├── isolinux.bin
│   ├── isolinux.cfg
│   ├── ldlinux.c32
│   ├── libcom32.c32
│   ├── libutil.c32
│   ├── menu.cfg
│   ├── splash.png
│   ├── stdmenu.cfg
│   └── vesamenu.c32
```

Better adding `config/includes.chroot/etc/skel/.nanorc`? What do you think?

karlinhigh committed:
May need Docker logout before build

I am new to Docker, and might not be understanding something here. I just installed Docker on Windows 10 Pro 1803 64-bit. My first effort at "docker-compose build" got me a message that said...

ERROR: Service 'lilypond-build' failed to build: Get https://registry-1.docker.io/v2/library/ubuntu/manifests/18.04: unauthorized: incorrect username or password

A web search advised me to log out my account from Docker, and the error was resolved. Maybe LilyDev's Docker instructions should mention this? Not sure which step would be best to include it.

karlinhigh committed:
Merge pull request #1 from karlinhigh/karlinhigh-patch-dockerlogout

May need Docker logout before build
